CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.2.3-SNAPSHOT] - 2023-03-20
+
+### Added
+
+ - ThreadService for storing different type of objects in HashMap stored in ThreadLocal
+ - Test for ThreadService
+
+### Changed
+
+ - IncomingDataAppResourceOverWs - WS flow can use the message handlers
```

Comment:
We should also think of way to "group" changes that are related with single task, so we know which changes goes with which issue, to have 'readable" changelog

Reply:
The only thing that comes to my mind is that we can link the description of added or changed things to a specific issue, but I think that isn't usual practice, so I think that a better approach could be that in a future version of CHANGELOG, we put the description in general, what has been done/changed, and not going into details about specific classes or methods.